File: examples-sdk/swift/001b-vilapp-ai-gw-benchmark/main.swift
// 001b-vilapp-ai-gw-benchmark — Swift SDK equivalent
// Compile: vil compile --from swift --input 001b-vilapp-ai-gw-benchmark/main.swift --release

let server = VilServer(name: "ai-gw-bench", port: 3081)
let gw = ServiceProcess(name: "gw")
server.service(gw)
server.compile()

File: examples-sdk/java/001b-vilapp-ai-gw-benchmark/Main.java
// 001b-vilapp-ai-gw-benchmark — Java SDK equivalent
// Compile: vil compile --from java --input 001b-vilapp-ai-gw-benchmark/Main.java --release

import dev.vil.*;

public class Main {
    public static void main(String[] args) {
        VilServer server = new VilServer("ai-gw-bench", 3081);
        ServiceProcess gw = new ServiceProcess("gw");
        server.service(gw);
        server.compile();
    }
}
